ethereum_history_api/oracles/src/script/prepareNoirFixtures.ts

```diff
@@ -1,16 +1,17 @@
-import { writeFile, mkdir, rm } from 'fs/promises';
+import { mkdir, rm, writeFile } from 'fs/promises';
 import { join } from 'path';
+import { Hex } from 'viem';
```

Comment:
Is there a logic to this new import order? If you've got something that sorts imports for you - would love to use it

Reply:
https://code.visualstudio.com/docs/languages/javascript#_organize-imports  `⇧⌥O` keyboard shortcut.
Remote deps first, local next, and alphabetical sort inside.